aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -133,9 +134,16 @@ class QConv2dInt8 final : public c10::OperatorKernel {
 
     fbgemm::DoNothing<> NoOpObj{};
 
-    auto bias_contig = bias.contiguous();
-    const auto* bias_ptr =
-        reinterpret_cast<int32_t*>(bias_contig.data<c10::qint32>());
+    const int32_t* bias_ptr = nullptr;
+    if (bias.has_value()) {
+      Tensor bias_vec = bias.value_or(Tensor());
```

Comment:
Here bias for sure has value, so you can directly call `bias.value()`

Reply:
Fixed. Thanks!